Task1/script.js

```diff
@@ -0,0 +1,39 @@
+const th = table.querySelectorAll('th');
```

Comment:
if you get some list of elements you need to name variable with `s`. for example `thElements`